curate/views_api.py

```diff
@@ -160,8 +160,23 @@ def list_articles(request):
     '''
     Return a list of all existing articles.
     '''
-    queryset=Article.objects.all().prefetch_related('commentaries')
-    serializer=ArticleListSerializer(instance=queryset, many=True)
+    PAGE_SIZE = 10
```

Comment:
Suggest accepting a `page_size` parameter rather than hardcoding to 10 articles per page.

Example:
https://github.com/ScienceCommons/curate_science/blob/ba08a2296c8163608be963f98a8ddea934f051a3/curate/views_api.py#L353